src/pymodaq/utils/parameter/pymodaq_ptypes/itemselect.py

```diff
@@ -84,7 +98,17 @@ def makeWidget(self):
         self.asSubItem = True
         self.hideWidget = False
         opts = self.param.opts
-        w = ItemSelect_pb()
+        
+        if 'checkbox' in opts and opts['checkbox']:      
+            w = ItemSelect_pb(checkbox = opts['checkbox'])
+            w.sigChanged = w.itemselect.itemChanged
```

Comment:
I guess this is taking care of my question above, but still need to be put in tests to ensure the behaviour is what we expect